src/wsl_tools/__init__.py

```diff
@@ -63,6 +62,22 @@ def from_dotdesktop(cls, app_def: Path) -> Optional[WSLApp]:
         icon = de.getIcon()
         if name:
             return cls(name, generic_name, cmd, gui, icon)
+        else:
+            # This is a symlink
+            linux_path = str(app_def)[str(app_def).index(r"\\wsl$") + 7:]
+            linux_path = linux_path[linux_path.index("\\"):].replace("\\", "/")
```

Comment:
We can extract this into a stand alone function, so that we can reuse it.
I also prefer to use `pathlib` instead of directly manipulate the path string, especially because this can be simply rewritten as: `linux_path = str(pathlib.PurePosixPath("/", *app_def.parts[1:]))`.